.github/workflows/_build.yml

```diff
@@ -256,3 +256,107 @@ jobs:
           git add app/screenshots/*
           git diff-index --quiet HEAD || git commit -m "Updated snapshots for '${{ matrix.variant }}'"
           git push
+
+  Screenshot:
+    name: Screenshots recording
+    runs-on: macos-latest
+    needs: CheckCommitMessage
+    if: ${{ needs.CheckCommitMessage.outputs.head-commit-message == 'Record screenshots' }}
```

Comment:
we should probably do this the same way we run the snapshot tests: always run them and check for changes and only commit with the trigger (and possibly use the same trigger as well to avoid having to wait twice). otherwise we risk those getting out of sync again